### PR TITLE
V2.0 dev

### PR DESCRIPTION
**Pull request by payten**

### CODE REVIEW

File: backend/model/ladybird_export.rb [hunk 1]
```diff
@@ -0,0 +1,693 @@
...
+
+  def to_stream
+    p = Axlsx::Package.new
+    wb = p.workbook
+
+    wb.add_worksheet(:name => 'Digitization Work Order') do |sheet|
+      highlight = sheet.styles.add_style :bg_color => "E8F4FF"
+
+      sheet.add_row column_definitions.collect{|col| col.fetch(:header)}
+      dataset.each do |row|
```
Comment (marktriggs) [inline backend/model/ladybird_export.rb]: Comment to mention the side-effect of setting up global data structures on this call

File: backend/model/ladybird_export.rb [hunk 2]
```diff
@@ -0,0 +1,693 @@
...
+        row_style = nil
+
+        if has_digital_object_instances?(row[:archival_object_id])
+          row_style = highlight
+        end
+
+        sheet.add_row column_definitions.map {|col| col[:proc].call(row, self) }, :style => row_style
```
Comment (payten) [inline backend/model/ladybird_export.rb]: Don't pass in `self`

File: backend/model/ladybird_export.rb [hunk 3]
```diff
@@ -0,0 +1,693 @@
...
+    @creators = {}
+    @resource_creators = {}
+    @agent_subjects = {}
+
+    subject_enum_id = nil
+    creator_enum_id = nil
+
+    EnumerationValue
+      .filter(:enumeration_id => Enumeration.filter(:name => 'linked_agent_role').select(:id))
+      .filter(Sequel.|({:value => 'creator', :value => 'subject' }))
```
Comment (payten) [inline backend/model/ladybird_export.rb]: Change to `.filter(:value => ['creator, 'subject'])`

File: backend/model/ladybird_export.rb [hunk 4]
```diff
@@ -0,0 +1,693 @@
...
+          node_to_position_map[row[:id]] = row[:position]
+          node_to_data_map[row[:id]] = row
+          node_to_root_record_map[row[:id]] = row[:root_record_id]
+          next_nodes_to_expand << row[:parent_id]
+        end
+
+        nodes_to_expand = next_nodes_to_expand.compact.uniq
+      end
+
+      ## Build up the path of waypoints for each node
```
Comment (marktriggs) [inline backend/model/ladybird_export.rb]: knock this out

File: backend/model/ladybird_export.rb [hunk 5]
```diff
@@ -0,0 +1,693 @@
...
+  end
+
+  def physical_description(row)
+    extents_for_archival_object(row[:archival_object_id])
+      .map{|row| "#{row[:number]} #{row[:extent_type]} (#{row[:portion]})" }
+      .join(NEW_LINE_SEPARATOR)
+  end
+
+  def name_subjects(row)
+    return if row[:archival_object_level] != 'item'
```
Comment (payten) [inline backend/model/ladybird_export.rb]: remove all `return if row[:archival_object_level] != 'item'` and show for all nodes

**Comment by jambun** [comment]
💋

**Pull request merged**